server/connector/serenedb_connector.hpp

```diff
@@ -153,7 +153,13 @@ class RocksDBTable final : public axiom::connector::Table {
     catalog::Column::Id col_id = 0;
     for (const auto& [name, type] : std::views::zip(
            collection.RowType()->names(), collection.RowType()->children())) {
-      auto column = std::make_unique<SereneDBColumn>(name, type, col_id++);
+      catalog::Column::Id id{};
+      if (name == catalog::Column::kFakeName) {
```

Comment:
What if someone will create a table with column named "fake"?
Like `CREATE TABLE foo (fake INTEGER PRIMARY KEY)` ?

Reply:
Use `"@fake"` now. It's invalid column name in SQL and PostgreSQL.

Reply:
There are some asserts on it, so just fixed it to deal with id